osmoutils/accum/accum.go

```diff
@@ -217,7 +219,7 @@ func (accum AccumulatorObject) AddToPositionCustomAcc(name string, newShares sdk
 // the unclaimed and newly accrued rewards and returns them alongside the redeemed shares. Then, it
 // overwrites the position record with the updated number of shares. Since it accrues rewards, it
 // also moves up the position's accumulator value to the current accum val.
-func (accum AccumulatorObject) RemoveFromPosition(name string, numSharesToRemove sdk.Dec) error {
+func (accum *AccumulatorObject) RemoveFromPosition(name string, numSharesToRemove sdk.Dec) error {
```

Comment:
Note: this is the core of the change that makes the receiver updated correctly.

This is important so that if we use the local variable after calling a mutative method on it, we can be confident that the local variable is updated